app/controllers/api/v1/documents_controller.rb

```diff
@@ -98,8 +98,10 @@ def decrypt_document_content
 
     def document_params
       params.require(:parameters)
-      d = params.require(:document)
-      d.require(:content)
+      params.require(:document).require(:content)
+
+      params[:parameters][:autoLoadEform] = true unless params[:parameters][:containerXmlns]
+
```

Comment:
Spravme na to aj test ak vieme

Reply:
momentálne nemáme ešte veľmi testy rozbehnuté na AVM. Bolo by to nadlhšie. Ten autoload pri nie-xml súboroch nerobí nič. Pri xml súboroch ho chceme mať iba vtedy, ak nie sú zadané custom parametre. containerXmlns je povinný parameter, ak podpisujeme eForm, takže podľa neho to vieme s istotou zistiť.